matlab/lib/+Ice/Exception.m

```diff
@@ -1,16 +1,7 @@
 classdef (Abstract) Exception < MException
     % Exception   Summary of Exception
     %
-    % Base class for Ice local and system exceptions.
-
+    % Base class for all Ice exceptions.
+    %
     % Copyright (c) ZeroC, Inc. All rights reserved.
-
-    methods(Abstract)
-        ice_id(obj)
```

Comment:
Moved ice_id for now to UserException. We most likely don't need it there either.